pages/in-vitro-systems/[id].js

```diff
@@ -37,11 +37,11 @@ import { isJsonFormat } from "../../lib/query-utils";
 
 export default function InVitroSystem({
   inVitroSystem,
-  biosampleTerm = null,
+  sampleTerms = null,
```

Comment:
This feedback applies to every file you changed under the "pages" directory. Since `sample_terms` and `sources` are required properties, and embedded, you don’t need to pass the `sampleTerms` and `sources` properties, since they’re already in `inVitroSystem` and the corresponding main properties on the other pages. Just use `inVitroSystem.sample_terms` and `inVitroSystem.sources` directly in this component. You can then also remove both `samplesTerms` and `sources` from PropTypes.

Reply:
Done. Thank you so much for explaining this!